api/sales_dashboard/views.py

```diff
@@ -93,12 +93,14 @@ def get_context_data(self, **kwargs):
         # when the caches were last updated.
         try:
             subscription_information_caches_influx_updated_at = (
-                OrganisationSubscriptionInformationCache.objects.order_by(
-                    "-influx_updated_at"
+                OrganisationSubscriptionInformationCache.objects.filter(
+                    influx_updated_at__isnull=False
                 )
+                .order_by("-influx_updated_at")
```

Comment:
If only have to get the first element, we can do ``desc(null_last=True)` /
docs: https://docs.djangoproject.com/en/3.1/ref/models/expressions/#django.db.models.Expression.desc

Reply:
Done.